examples/opencv_rc.py

```diff
@@ -0,0 +1,541 @@
+        Toggle the state of the light.
+        :param value: Bool. True for turning the light on, False otherwise.
+        :return: None
+        """
+
+        if self._head_light != value:
+            # Set the value
+            self._head_light = value
+
+            # Tell Cozmo to actually turn the light on/off
+            self._cozmo_clt.set_head_light(enable=self._head_light)
+
+    @property
+    def head_tilt(self):
+        return self._head_tilt
+
+    @head_tilt.setter
+    def head_tilt(self, value):
+        """
+        Set the head tilt and tell Cozmo to actually move its head.
+        :param value: Float. The new head angle in radians.
+        :return: None
+        """
+
+        # Set the value of the head tilt
+        self._head_tilt = max(pc.MIN_HEAD_ANGLE.radians,
+                              min(pc.MAX_HEAD_ANGLE.radians, value))
+
+        # Send the command to cozmo
+        self._cozmo_clt.set_head_angle(self._head_tilt)
+
+    @property
+    def lift_height(self):
+        return self._lift_height
+
+    @lift_height.setter
+    def lift_height(self, value):
+        """
+        Set the lift height and tell Cozmo to actually move the lift.
+        :param value: Float. The new horizontal position of the lift in mm.
+        :return: None
+        """
+
+        # Set the value of the lift height
+        self._lift_height = max(pc.MIN_LIFT_HEIGHT.mm,
+                                min(pc.MAX_LIFT_HEIGHT.mm, value))
+
+        # Send the command to cozmo
+        self._cozmo_clt.set_lift_height(height=self._lift_height)
+
+    @property
+    def linear_velocity(self):
+        return self._velocity['linear']
+
+    @linear_velocity.setter
+    def linear_velocity(self, value):
+        self._velocity['linear'] = max(0, min(value, pc.MAX_WHEEL_SPEED.mmps))
+
+    @property
+    def angular_velocity(self):
+        return self._velocity['angular']
+
+    @angular_velocity.setter
+    def angular_velocity(self, value):
+        self._velocity['angular'] = max(0,
+                                        min(value, pc.MAX_WHEEL_SPEED.mmps /
+                                            pc.TRACK_WIDTH.mm))
+
+    @property
+    def color(self):
+        return self._color
+
+    @color.setter
+    def color(self, value):
+        """
+        Switch between colored and grayscale image for Cozmo's video feed.
+        :param value: Bool. True displays a colored image, a grayscale one
+        otherwise.
+        :return: None
+        """
+
+        if self._color != value:
+            # Set the new value for the color switch
+            self._color = value
+
+            # Tell cozmo to actually change the image
+            self._cozmo_clt.enable_camera(enable=True, color=value)
+
+
+if __name__ == "__main__":
+    # Instantiate a new remote controller
+    rc = OpencvRC()
+
+    # Initialize the remote controller
+    rc.init()
+
+    try:
+        # Instantiate a dummy event
+        evt = Event()
```

Comment:
I don't think you need this. You are creating an event to do the work of sleep. I think you can do better ;)

Reply:
You are right. If I put the display function back into the main thread, this will be useless. I did not like it anyway, so good ridance.